org.msweet.htmldoc.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.msweet.htmldoc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: htmldoc
+rename-icon: htmldoc
+rename-desktop-file: htmldoc.desktop
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  #- --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib64/pkgconfig
+  - /lib/cmake
+modules:
+  - name: fltk
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://www.fltk.org/pub/fltk/1.3.8/fltk-1.3.8-source.tar.bz2
+        sha256: 1d9832409fc08c908c64ab40ad9fd8b0e8af5d203651eb9690b810f41e2f412d
+  - name: htmldoc
+    buildsystem: autotools
+    post-install:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo org.msweet.htmldoc.metainfo.xml
+      - install -Dm644 desktop/htmldoc-128.png ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/org.msweet.htmldoc.png
```

Comment:
```suggestion
```

See my other suggestion in the metainfo. It is the typo in the desktop file name that caused `appstream-compose` to not find the icons. (this is when the get copied)

Reply:
Just to add, the icons are in the right place as they are.

Reply:
OK, that does explain things... Thanks!